packages/ses/src/get-anonymous-intrinsics.js

```diff
@@ -168,17 +168,9 @@ export const getAnonymousIntrinsics = () => {
   }
 
   const ab = new ArrayBuffer(0);
-  let iab;
-  // @ts-expect-error see below
-  if (typeof ab.transferToImmutable === 'function') {
-    // @ts-expect-error TODO How do I add transferToImmutable to ArrayBuffer type?
-    // eslint-disable-next-line @endo/no-polymorphic-call
-    iab = ab.transferToImmutable(); // Note: Unavailable on Hermes
-  } else {
-    // @ts-expect-error TODO How do I add sliceToImmutable to ArrayBuffer type?
-    // eslint-disable-next-line @endo/no-polymorphic-call
-    iab = ab.sliceToImmutable();
-  }
+  // @ts-expect-error TODO How do I add sliceToImmutable to ArrayBuffer type?
+  // eslint-disable-next-line @endo/no-polymorphic-call
+  const iab = ab.sliceToImmutable();
```

Comment:
No reason for the condition. `sliceToImmutable` works fine for both.

Reply:
was thinking exactly this, nice